.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I don't think it is good idea to add vscode settings into repository. Please remove this file/settings to keep it local.